tools/static-users-generator/scripts/config_replace.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
your team really like coding in bash?
I still have the same opinion about that as previously...
https://github.com/kyma-project/kyma/pull/861#discussion_r220844279

Reply:
That is not about what we like. The approach with bash script was faster to implement in case of such a small tool. We will be probably implementing this tool in go later.